physionet-django/console/templates/console/published_projects.html

```diff
@@ -11,7 +11,7 @@
 {% block content %}
 <div class="card mb-3">
   <div class="card-header">
-    Published Projects <span class="badge badge-pill badge-info">{{ projects|length}}</span>
+    Published Projects <span class="badge badge-pill badge-info">{{ projects.paginator.count}}</span>
```

Comment:
While you're updating the thing above, please could you add a space between the variables and the `}}` template tags for consistency?